electron_app/helpers/handleDisk.js

```diff
@@ -0,0 +1,19 @@
+const fs = require('fs');
+const mkdirp = require('mkdirp');
+
+const handleFolder = (directory) => {
+    return new Promise((resolve, reject) =>{
+        if (!fs.existsSync(directory)) {
+
+            mkdirp.sync(directory, (err) => {
+                if (err) {
+                    console.log(err);
```

Comment:
You should test this, but I don't think `console.log()` will work in this context. It's for debugging only, since we'll need user-facing error reporting for this error, but if it doesn't work it will be confusing if it fails but doesn't produce a message.